PWGHF/HFC/Tasks/taskCorrelationD0Hadrons.cxx

```diff
@@ -503,7 +503,7 @@ struct HfTaskCorrelationD0Hadrons {
       float bdtScorePromptD0bar = pairEntry.mlScorePromptD0bar();
       float bdtScoreBkgD0bar = pairEntry.mlScoreBkgD0bar();
       bool isPhysicalPrimary = pairEntry.isPhysicalPrimary();
-      int statusD0Prompt = static_cast<int>(pairEntry.isPrompt());
+      bool statusD0Prompt = pairEntry.isPrompt();
```

Comment:
Adjust the name according to the change of type and meaning. `statusD0Prompt` -> `isD0Prompt`.